roles/openshift_node/tasks/create_machine.yml

```diff
@@ -3,13 +3,12 @@
 
 - name: Create OpenShift object
   command: >
-    oc create -f {{ gpu_machineset_deff }}
-    --kubeconfig={{ kubeconfig }}
+    bash -c "oc create -f {{ gpu_machineset_deff }} --kubeconfig '{{ kubeconfig }}'"
```

Comment:
@ArangoGutierrez could you quote this one as well, please?

Reply:
done